README.md

```diff
@@ -316,48 +390,73 @@ logReturner();
+Ditto for the `console` object's `warn()` method, which provides an intermediate
+step between a regular `log()` message and a more dire `error()` message and is
+the source of those bright yellow messages you'll occasionally see in the
+browser's JS console:
 
 ![`console.warn()`](https://curriculum-content.s3.amazonaws.com/web-development/js/basics/intro-to-debugging-readme/console_error_log_warn.png)
 
-The `warn()` method is often used to alert developers looking at the JS console that they've done something inadvisable but not reaching the level of a full-blown error. For example, we could highlight the usage of a soon-to-be-deprecated feature:
+The `warn()` method is often used to alert developers looking at the JS console
+that they've done something inadvisable but not reaching the level of a
+full-blown error. For example, we could highlight the usage of a
+soon-to-be-deprecated feature:
 ```js
 console.warn("Hmmm, you probably don't want to do that...");
 // WARN: Hmmm, you probably don't want to do that...
 ```
 
-When we use `console.warn()` in code snippets, we'll preface the output statements with `WARN:` to differentiate them from other logged messages.
+When we use `console.warn()` in code snippets, we'll preface the output
+statements with `WARN:` to differentiate them from other logged messages.
 
 ## Conclusion
-Over the course of your programming career, you'll probably spend **significantly** more time debugging than actually writing new code. Just as your coding skills will improve with practice, so too will your debugging skills.
-
-Debugging can sometimes feel very demoralizing. You'll fix one bug and ten new ones appear:
-
-<picture>
-  <source srcset="https://curriculum-content.s3.amazonaws.com/web-development/js/basics/intro-to-debugging-readme/bugs.webp" type="image/webp">
-  <source srcset="https://curriculum-content.s3.amazonaws.com/web-development/js/basics/intro-to-debugging-readme/bugs.gif" type="image/gif">
-  <img src="https://curriculum-content.s3.amazonaws.com/web-development/js/basics/intro-to-debugging-readme/bugs.gif" alt="WHERE'S MY MAC N' CHEESE?">
-</picture>
-
-If it's any `console`-ation, we **all** make mistakes. Treat debugging as a learning opportunity. Often, looking at your code critically and trying to figure out why something isn't working will afford you a much deeper understanding of how some feature of the language actually works. We'll continue to use the `console` object and other tools throughout this course. By the end, you'll be on your way to being a debugging master!
+Over the course of your programming career, you'll probably spend
+**significantly** more time debugging than actually writing new code. Just as
+your coding skills will improve with practice, so too will your debugging
+skills.
+
+Debugging can sometimes feel very demoralizing. You'll fix one bug and ten new
```

Comment:
`demoralizing` to `tedious`, new devs will develop low self-esteem on their own 😉